telegram/ext/_dispatcher.py

```diff
@@ -631,6 +631,40 @@ def remove_handler(self, handler: Handler, group: int = DEFAULT_GROUP) -> None:
             if not self.handlers[group]:
                 del self.handlers[group]
 
+    def migrate_chat_data(
+        self, message: 'Message' = None, old_chat_id: int = None, new_chat_id: int = None
+    ) -> None:
+        """
+        Migrate chat_data content
```

Comment:
Moves the contents of :attr:\`chat_data\` at key ``old_chat_id`` to the key ``new_chat_id``. Also updates the persistence by calling :attr:\`update_persistence\`.